src/features/DisplayControls/NursingTasks/components/NursingTasks.jsx

```diff
@@ -253,132 +253,186 @@ export default function NursingTasks(props) {
-                ? setFilterValue(event.selectedItem)
-                : setFilterValue(items[2]);
-            }}
-          />
-          <Button
-            kind={"tertiary"}
-            isExpressive
-            size="default"
-            renderIcon={Add16}
-            onClick={() => {
-              if (!isSliderOpen.emergencyTasks) {
-                updateEmergencyTasksSlider(true);
-              }
-            }}
-          >
-            <FormattedMessage id={"ADD_TASK"} defaultMessage={"Add Task"} />
-          </Button>
+      );
+    } else {
+      return (
+        <div className="shift-timing">
+          <div className="shift-timing-title">
+            <FormattedMessage
+              id={"SHIFT_TIMING"}
+              defaultMessage={"Shift timing"}
+            />
+          </div>
+          <div className="shift-time">
+            {shiftStartDate} | <Time16 /> {shiftStartTime} to {shiftEndDate} |{" "}
+            <Time16 /> {shiftEndTime}
+          </div>
         </div>
-      </div>
+      );
+    }
+  };
 
-      {isSliderOpen.nursingTasks && (
-        <UpdateNursingTasks
-          medicationTasks={selectedMedicationTask}
-          updateNursingTasksSlider={updateNursingTasksSlider}
-          patientId={patientId}
-          providerId={provider.uuid}
-          setShowSuccessNotification={setShowSuccessNotification}
-          setSuccessMessage={setSuccessMessage}
-        />
-      )}
-      {isSliderOpen.emergencyTasks && (
-        <AddEmergencyTasks
-          patientId={patientId}
-          providerId={provider.uuid}
-          updateEmergencyTasksSlider={updateEmergencyTasksSlider}
-          setShowSuccessNotification={setShowSuccessNotification}
-          setSuccessMessage={setSuccessMessage}
-        />
-      )}
+  return (
+    <div className="nursing-tasks-content-container display-container">
       {isLoading ? (
-        <div className="loading-parent" data-testid="loading-icon">
-          <Loading withOverlay={false} />
-        </div>
-      ) : medicationNursingTasks && medicationNursingTasks.length === 0 ? (
-        <div className="no-nursing-tasks">{getNoTaskMessage()}</div>
-      ) : (
-        <div>
-          <div className="nursing-task-tiles-container">{showTaskTiles()}</div>
-          <AdministrationLegend />
-        </div>
-      )}
-      {showSuccessNotification && (
-        <Notification
-          hostData={{
-            notificationKind: "success",
-            messageId: successMessage,
-          }}
-          hostApi={{
-            onClose: () => {
-              setShowSuccessNotification(false);
-              refreshDisplayControl([
-                componentKeys.TREATMENTS,
-                componentKeys.NURSING_TASKS,
-                componentKeys.DRUG_CHART,
-              ]);
-            },
-          }}
-        />
-      )}
-    </div>
-  );
-}
+          <div style={{ paddingTop: "5px" }} data-testid="loading-icon">Loading...</div>
```

Comment:
this is not needed now, we already have this condition below... Please check the latest code from main